pkgs/development/python-modules/paramiko/default.nix

```diff
@@ -18,12 +18,12 @@
 
 buildPythonPackage rec {
   pname = "paramiko";
-  version = "3.4.1";
+  version = "3.5.0";
   format = "setuptools";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-ixUwKHCvf2ZS8uA4l1wdKXPwYEbLXX1lNVZos+y+zgw=";
+    hash = "sha256-rRHlQNpPVc7dpSkx8aP4Eqgjinr39ipg3lOM2AuygSQ=";
   };
 
```

Comment:
Could you remove `with lib;` in line 68? (ref: https://github.com/NixOS/nixpkgs/issues/208242)
You can use `lib.licenses.lgpl21Plus` instead of `licenses.lgpl21Plus`.